lib/cli/Cargo.toml

```diff
@@ -40,6 +40,8 @@ wasmer-types = { version = "=3.1.0", path = "../types" }
 wasmer-registry = { version = "=3.1.0", path = "../registry" }
 wasmer-object = { version = "=3.1.0", path = "../object", optional = true }
 wasmer-vfs  = { version = "=3.1.0", path = "../vfs", default-features = false, features = ["host-fs"] }
+wasmer-wasm-interface = { version = "3.0.1", path = "../wasm-interface" }
```

Comment:
isn't the version supposed to be "=3.1.0"?

Reply:
yeah thanks for catching